app/Bus/Commands/ComponentGroup/UpdateComponentGroupCommand.php

```diff
@@ -49,31 +49,49 @@
     public $collapsed;
 
     /**
+     * To whom should the component group be visible?
+     *
+     * @var int
+     */
+    public $visible;
+
+    /**
+     * The id of the creator of the component group.
```

Comment:
Missing a `var` doc here. Also, is there a reason why this couldn't be the currently logged in user?